docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -72,17 +76,121 @@ \section{Critical Assumptions}
+		 & \multicolumn{1}{|c|}{\textbf{Effects of Failure}}
+		 & \multicolumn{1}{|c|}{\textbf{Causes of Failure}}
+		 & \multicolumn{1}{|c|}{\textbf{Recommended Action}}
+		 & \multicolumn{1}{|c|}{\textbf{SR}}
+		 & \multicolumn{1}{|c|}{\textbf{Ref.}}                                                                          \\
+		\hline
+		\multirow{2}{*}{Work Orders}
+		 & Work Order is missing
+		 & Customer and Employees will not know any of the work that has been done for a specific job
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Regular and automatic database backups/snapshots and allow shop owners to request rollbacks
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H5-1                                                                                                         \\
+		\cline{2-7}
+		~
+		 & Work Order is missing detailed information
+		 & Customer and Employees will not know all of the work that has been done for a specific job
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Refer to H5-1a
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H5-2                                                                                                         \\
+		\hline
+		Shop Profile
+		 & Unable to find details of a shop
+		 & Customers will not be able to see the information about a specific shop (e.g., address, phone number, etc.)
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Refer to H5-1a
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H6-1                                                                                                         \\
+		\hline
+		\multirow{2}{*}{Services}
+		 & Unable to find a service
+		 & Customers and Employees will not be able to see what services are offered by the shop
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Refer to H5-1a
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H7-1                                                                                                         \\
+		\cline{2-7}
+		~
+		 & Unable to find details of a service
+		 & Customers and Employees will not be able to see the details of a service (e.g., price, estimated time, etc.)
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
```

Comment:
could this be due to another failure? e.g., a bug?

Reply:
https://github.com/arkinmodi/project-sayyara/pull/144#discussion_r997616169